CHANGELOG.md

```diff
@@ -1 +1,2 @@
 - Added rudimentary email enumeration protection for auth emulator. (#6702)
+- You can now run customized automated tests on your Android apps in App Distribution, with the Automated Tester feature (beta). This feature automatically runs tests on your Android apps on virtual and physical devices at different API levels. To learn how to run an automated test, see [Run an automated test for Android apps](https://firebase.google.com/docs/app-distribution/android-automated-tester). (#6730)
```

Comment:
Question for the Firebase CLI reviewer - we heard there might be a way to mark something as "beta" in the Firebase CLI. Is this applicable here?

Reply:
Nvm - we found go/firebase-cli-contributor#hide-the-command-behind-a-flag, but given this is not a new command (just a new optional param on an existing command), it's not applicable here